### DISCUSSION TITLE
Higher GPS update frequency

### DISCUSSION THREAD

**Discussion by ghost** [Zverik/every_door#805] [General]
The GPS update frequency is very low and I have to wait way too long to see how good the position is when adding points. 
The update frequency should be as high as possible, like 1 per second. 
It would also helpful if the app has a function to average the position for a while if desired. Locus Map has a function like that.

**Comment by ghost** [comment]
Another point: Other GPS apps show a circle with the size of the GPS accuracy. That would make it easier to judge the current accuracy at a glance.
Every Door really needs to add a way to improve GPS accuracy when mapping, since everything in OSM depends on GPS coordinates.

**Comment by deevroman** [comment]
Just in case, what version of Every Door do you have? In 5.1 the refresh rate is increased to 1 second https://github.com/Zverik/every_door/commit/a9cf8287817a944b15baa2324b6a356416e14f97 However, as I understand it, there is a certain distance threshold after which the current position is updated

**Comment by ghost** [comment]
5.2, but I don't know what the version was 2 weeks ago when I made a note to discuss this here.  
A distance threshold makes sense for a GPS tracker, but not if you are stationary and want to pinpoint the current location, like in Every Door.

Just checked it in the app being stationary. Time between updates is exactly 10 seconds, not 1. I don't see any distance threshold being in effect.
